playbooks/prepare_host.yaml

```diff
@@ -2,20 +2,42 @@
 - hosts: standalone
   become: true
   become_user: root
-  gather_facts: false
+  gather_facts: true
   vars_files: vars/defaults.yaml
 
   tasks:
 
-  - name: install rhos-release
-    yum:
-      state: installed
-      name:
-      - http://download.devel.redhat.com/rcm-guest/puddles/OpenStack/rhos-release/rhos-release-latest.noarch.rpm
-      disable_gpg_check: True
+  - name: Prepare host on RHEL system
+    when:
+      - ansible_facts.distribution == 'RedHat'
+    block:
+      - name: install rhos-release
+        yum:
+          state: installed
+          name:
+          - http://download.devel.redhat.com/rcm-guest/puddles/OpenStack/rhos-release/rhos-release-latest.noarch.rpm
+          disable_gpg_check: True
+
+      - name: Configure rhos release "{{ rhos_release }}"
+        shell: rhos-release "{{ rhos_release }}"
+
+      - name: Fetch the Red Hat root certificate
```

Comment:
I think we also need this when pulling directly from internal registries.

However... that's dependent on where we run the installer from, so I'm ok with ditching this here and adding it back anywhere else it's needed when we find it, perhaps after refactoring it into a role.

Reply:
let's keep it specific to RHEL now and I'm happy to fix it later if we hit the issue.